docs/docs/sticky-bucketing.mdx

```diff
@@ -30,16 +30,34 @@ Sticky Bucketing is a GrowthBook Pro and Enterprise feature.
 
 :::
 
-## Setting up Sticky Bucketing
+## Setting up Sticky Bucketing with the Script Tag SDK
+
+### 1. Configure the script tag
+
+Using Sticky Bucketing with our [HTML Script Tag](https://docs.growthbook.io/lib/script-tag) is simple. Simply add either of these properties to your script tag:
```

Comment:
We repeat "simple. Simply" I'd take out the second one: "Add either of these attributes to your script tag:"